cmd/cogito/main_test.go

```diff
@@ -79,8 +79,10 @@ func TestRunPutSuccess(t *testing.T) {
 	}))
 	var out bytes.Buffer
 	var logOut bytes.Buffer
+
+	gitHubSpyDomain, _ := url.Parse(gitHubSpy.URL)
 	inputDir := testhelp.MakeGitRepoFromTestdata(t, "../../cogito/testdata/one-repo/a-repo",
-		testhelp.HttpsRemote("the-owner", "the-repo"), "dummySHA", wantGitRef)
+		testhelp.HttpsRemote(gitHubSpyDomain.Host, "the-owner", "the-repo"), "dummySHA", wantGitRef)
```

Comment:
GOAL: avoid always creating a `.git/config` file with the domain `github.com` hardcoded. This makes all the tests passing because we had: `left := []string{"github.com", owner, repo}` in putter hardcoded also. This is a bit wrong because in the test we override the GitHub endpoint, so we should be able to propagate this value.